Source/Core/DolphinWX/NetWindow.cpp

```diff
@@ -97,11 +97,47 @@ static void FillWithGameNames(wxListBox* game_lbox, const CGameListCtrl& game_li
 NetPlaySetupDiag::NetPlaySetupDiag(wxWindow* const parent, const CGameListCtrl* const game_list)
 	: wxFrame(parent, wxID_ANY, _("Dolphin NetPlay Setup"))
 	, m_game_list(game_list)
+	, m_central_server("")
+	, m_central_port(0)
 {
 	IniFile inifile;
 	inifile.Load(File::GetUserPath(D_CONFIG_IDX) + "Dolphin.ini");
 	IniFile::Section& netplay_section = *inifile.GetOrCreateSection("NetPlay");
 
+	// Check traversal ini
```

Comment:
That's a very weird way to handle settings compared to how we do it for everything else.